MBBSEmu/HostProcess/ExportedModules/Galgsbl.cs

```diff
@@ -24,6 +24,8 @@ public class Galgsbl : ExportedModuleBase, IExportedModule, IStoppable
         /// <returns></returns>
         public const ushort Segment = 0xFFFE;
 
+        public const ushort OUTBUF_SIZE = 8192;
```

Comment:
We can move this constant to the base class so the ExportedModules don't reference one another